.travis.yml

```diff
@@ -0,0 +1,35 @@
+language: php
+
+php:
+  - 5.4
+  - 5.5
+  - 5.6
+  - 7.0
+  - hhvm
+  - nightly
+
+sudo: false
+
+cache:
+  directories:
+    - $HOME/.composer/cache/files
+
+matrix:
```

Comment:
File is smaller, and travis set's env before running job. I had travis error (that he can't set php env) on some jobs with this (current) way.

And at end you have allways testsed only --prefer-stable (as you have two flags at once). Just check how travis build looks - https://travis-ci.org/SuperdeskWebPublisher/SWPMultiTenancyBundle/builds/105408330. It's all wrong. 

We should tests every php with --prefer-lowest and --prefer-stable (jobs without flags are using allways --prefer-stable as defaults - as we don't have lock file here).